docs/1-Applying/finding_companies_to_apply_to.md

```diff
@@ -4,30 +4,36 @@
-- https://www.indeed.com/
-- https://breakoutlist.com/all
-- https://www.wayup.com/
+- [Intern Supply](http://www.intern.supply/) (links you directly to the application page of a lot of companies)
+- [Research in the US](https://www.nsf.gov/crssprgm/reu/list_result.jsp?unitid=5049)
+- [Indeed.com](https://www.indeed.com/) (job board)
+- [Breakout List](https://breakoutlist.com/all) (hot new tech companies)
+- [WayUp](https://www.wayup.com/) (internships)
+- [Hacker News](https://news.ycombinator.com/) (for the latest monthly list, search for "Ask HN: Who is hiring? (<month> <year>)". For example, "Ask HN: Who is hiring? (February 2017)".)
 - TODO: add some links to lists people often use (idk, can some people recommend? I haven't used these much)
 
 ### Join Facebook groups (and mailing lists, slack channels, whatever is out there) that have job postings.
 
-- [https://www.facebook.com/groups/HHinternships/?ref=br_rs](https://www.facebook.com/groups/HHinternships/?ref=br_rs)
-- [https://www.facebook.com/groups/733654363389653/?ref=br_rs](https://www.facebook.com/groups/733654363389653/?ref=br_rs)
-- [https://www.facebook.com/groups/ncwandintern/?ref=br_rs](https://www.facebook.com/groups/ncwandintern/?ref=br_rs)
+- [Hackathon Hackers Internships](https://www.facebook.com/groups/HHinternships)
+- [Hackathon Hackers Internships/Co-ops](https://www.facebook.com/groups/733654363389653)
+- [Bay Area Jobs Network](https://www.facebook.com/groups/ncwandintern)
 - etc... you should also check out intern groups relevant to your school! they probably exist
 
-
 ### Google it
 
-Sometimes I just google "hiring interns" (usually adding a preferred location and/or an industry I'd like to work in) and see what shows up! This works pretty well too
+Information about available opportunities is scattered throughout the Internet moves around a lot. Sometimes I just google "hiring interns" (adding a few more specific terms) and see what shows up! This works pretty well too.
+
+The more specific the query, the more likely you can find something useful. Try different combinations of queries that include some of the below:
+- What you're looking for. It can be "tech company internships", "hiring programming intern", etc. Since not all companies have internship positions advertised, you may want to just search for "tech company" and manually find someone to contact on their website.
+- The specific industry you are looking for. For example, "biotech company", "gaming studio", "app development shop", "data analytics provider", etc.
+- A specific city. Beyond the Bay Area, there are many smaller tech hubs in the US and Canada. You can search for internships in <your city>, <a big city near you>, New York, Seattle, Boston, Portland, Austin, Pittsburg, Boulder, Washington, Waterloo, Toronto, Vancouver, Montreal. Outside of North America, tech hubs include London, Dublin, Berlin, Amsterdam, Singapore, Taipei, Hong Kong, and probably a lot more that I don't know.
```

Comment:
love the list - maybe even explicitly say that internships are a great opportunity for travel and seeing new places